### PR TITLE
Some very minor changes to the first chapters

### PR DESCRIPTION
**Pull request by nrc**
Some grammar fixes, some indentation fixes to make it easier to copy and paste code, and a few changes to try and make things clearer.

### CODE REVIEW

File: content/docs/getting-started/hello-world.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Let's get started.
 First, generate a new crate.
 
 ```bash
-$ cargo new --bin hello-world
```
Comment (nrc) [inline content/docs/getting-started/hello-world.md]: `--bin` is the default for quite a while now

File: content/docs/getting-started/hello-world.md [hunk 2]
```diff
@@ -203,7 +203,7 @@ You can find the full example [here][full-code].
 command starts a listening TCP socket on the previously specified port.
 
 ```bash
-$ nc -l -p 6142
+$ nc -l 6142
```
Comment (nrc) [inline content/docs/getting-started/hello-world.md]: I'm not 100% sure about this, but on Mac OS at least, `-p` causes an error and netcat works without it. I haven't tested on other platforms but this does correspond to some documentation I found.

**Comment by carllerche** [comment]
Thanks, these look good.

Unfortunately, there are conflicts. Instead of rebasing, I'll try including these changes as part of #241.

**Comment by rylev** [comment]
@carllerche @nrc Should we close this? Has this been integrated into #241?

**Comment by carllerche** [comment]
I need to double check...

**Comment by carllerche** [comment]
These changes were applied in #263

Thanks for the original PR <3

**Pull request closed without merging**